index.js

```diff
@@ -4,19 +4,62 @@
 
 var arguments = process.argv.slice(2);
 
-var operand1 = arguments[0];
-var operand2 = arguments[1];
+var operand1 = toNumber(arguments[0], 10);
+var operand2 = toNumber(arguments[1], 10);
 var operator = arguments[2];
+var result = 0;
 
 console.log('Calculating:', operand1, operand2, operator);
 
-var result;
+calculateSetOperation();
 
-switch (operator) {
-  case '+':
-    result = operand1 + operand2;
-  default:
-    console.error('Not implemented:', operator);
+// selects the correct case and performs the correct calculation depending on the operands
+function calculateSetOperation() {
+
+  switch (operator) {
+    case '+':
+      result = addition(operand1, operand2);
+      break;
+    case '-':
+      result = subtraction(operand1, operand2);
+      break;
+    /*
+    case '*':
+      result = multiplication(operand1, operand2);
+      break;
+      case '/':
+        result = division(operand1, operand2);
+        break;
+    */
+    default:
+      console.error('Not implemented:', operator);
+  }
+  console.log(result);
+}
+
+// returns the parsing from a string to a number
+function toNumber(x) {
```

Comment:
change to take into account comma numbers

Reply:
👍